### DISCUSSION TITLE
docs.getoutline.com requires email login

### DISCUSSION THREAD

**Discussion by binaryfire** [outline/outline#6610] [General] [answered]
Hi @tommoor

I just tried to look at the docs at [docs.getoutline.com](https://docs.getoutline.com/) and see they require authentication. Is that a mistake or are they only accessible to paid customers now?

<img width="759" alt="image" src="https://github.com/outline/outline/assets/5361908/9b049f7d-96a3-458a-8fe6-223ff33aa045">

**Comment by JairoGuo** [comment]
I'm having this problem too. Yesterday it was fine.

**Comment by johnny-smitherson** [comment]
Ah, this is not intentional?

**Reply by binaryfire** [reply]
I've been self-hosting Outline for a couple of years and the docs never used to require logging in.

**Comment by tommoor** [comment, marked as answer]
This was a mistake, the guides are restored now

**Reply by johnny-smitherson** [reply]
<img width="907" alt="aaCapture" src="https://github.com/outline/outline/assets/127537716/fa56266a-0e27-4c91-a105-f578483d4dfb">

**Reply by tommoor** [reply]
They have never been available at the root domain, it's https://docs.getoutline.com/s/hosting
